dev-utils/src/commands/genesis_command.rs

```diff
@@ -18,84 +25,268 @@ use crate::commands::genesis_dev_config::GenesisDevConfig;
 
 #[derive(Debug, StructOpt)]
 pub enum GenesisCommand {
-    #[structopt(
-        name = "generate",
-        about = "Generate new genesis.json parameters (app_hash + app_state)"
-    )]
+    #[structopt(name = "generate", about = "Generate new genesis.json")]
     Generate {
+        #[structopt(
+            name = "tendermint_genesis_path",
+            short,
+            long,
+            help = "Path to the Tendermint genesis.json file (e.g. ~/.tendermint/config/genesis.json)"
+        )]
+        tendermint_genesis_path: Option<PathBuf>,
+
         #[structopt(
             name = "genesis_dev_config_path",
             short,
             long,
             help = "Path to a file containing the genesis-related configuration (e.g. ERC20 holdership) -- see example-dev-conf.json"
         )]
         genesis_dev_config_path: PathBuf,
+
+        #[structopt(
+            name = "inline",
+            short,
+            long,
+            help = "Replace Tendermint genesis.json file in place, saving backups with .bak.json extension"
+        )]
+        inline: Option<bool>,
```

Comment:
No need to have `Option<bool>`. Only `bool` should work.

See this: https://github.com/crypto-com/chain/blob/1e3485c5bf3f9b83565878af314d8858213f88c7/client-cli/src/command.rs#L105

Reply:
Thanks! I notice `bool` does not need Optional before but I check the `structopt` they mention optional field is denoted by adding `Option`. That's why I add it. I will remove the unnecessary `Option`

Reply:
Done